README.md

```diff
@@ -1,3 +1,8 @@
+# Warning: this project as been deprecated
```

Comment:
There is a non zero-chance the Vercel build is failing because of the two top-level headers / H1s?

Maybe move it below the `# svelte-cubed` section, and also fix the little typo? 🙂 

```suggestion
## Warning: this project has been deprecated
```

Great idea raising this, I've read that discussion thread and still didn't quite remember which is the more supported library now 😅